### PR TITLE
Feature/account settings security (Password update) 

### PR DESCRIPTION
**Pull request by leomendoza123**

### CODE REVIEW

File: src/app/account-settings/components/settings-security/settings-security.component.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<h2 class="orc-font-body col" i18n="@@account.defaults">Security</h2>
```
Comment (DanielPalafox) [inline src/app/account-settings/components/settings-security/settings-security.component.html:1]: @leomendoza123 Can you fix this title

File: src/app/account-settings/components/settings-security/settings-security.component.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit } from '@angular/core'
+
+@Component({
+  selector: 'app-settings-security',
+  templateUrl: './settings-security.component.html',
+  styleUrls: ['./settings-security.component.scss'],
+})
+export class SettingsSecurityComponent implements OnInit {
+  titleAccountPassword = $localize`:@@account.emailFrequency:Account password  `
```
Comment (DanielPalafox) [inline src/app/account-settings/components/settings-security/settings-security.component.ts:9]: @leomendoza123 Can you fix this localize property

File: src/app/account-settings/components/settings-security-password/settings-security-password.component.html [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    <mat-error *ngFor="let error of errors" class="error-container mat-caption">
+      {{ error }}
+    </mat-error>
+  </ng-container>
+
+  <div *ngIf="success === true" class="error-container">
+    Your password has been updated
+  </div>
+
+  <div class="button-container" (click)="save()">
```
Comment (DanielPalafox) [inline src/app/account-settings/components/settings-security-password/settings-security-password.component.html:136]: @leomendoza123 click event save it should go in the button not in the div

File: src/app/account-settings/components/settings-security-password/settings-security-password.component.html [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+    </mat-error>
+  </mat-form-field>
+
+  <ng-container *ngIf="!success">
+    <mat-error *ngFor="let error of errors" class="error-container mat-caption">
+      {{ error }}
+    </mat-error>
+  </ng-container>
+
+  <div *ngIf="success === true" class="error-container">
```
Comment (DanielPalafox) [inline src/app/account-settings/components/settings-security-password/settings-security-password.component.html:132]: @leomendoza123 i18 property is missing here

**Review by DanielPalafox — CHANGES_REQUESTED** [review]

**Review by amontenegro — APPROVED** [review]

**Pull request merged**